Source/Scene/OctahedralProjectedCubeMap.js

```diff
@@ -302,11 +303,22 @@ OctahedralProjectedCubeMap.prototype.update = function (frameState) {
     return;
   }
 
+  var defines = [];
+  if (
```

Comment:
Still to be tested with the SFLOAT formats from the IBL sampler but I imagined that the new `isNormalizedFormat` function would catch both when pushing the define.